examples/widgetbook_example/widgetbook/widgetbook.dart

```diff
@@ -47,6 +41,15 @@ class HotreloadWidgetbook extends StatelessWidget {
               WidgetbookTheme(name: 'Dark', data: darkTheme),
             ],
           ),
+          frameSetting: FrameSetting.firstAsSelected(frames: [
+            DeviceFrameFrame(
```

Comment:
We should rename this to something that's not called `DeviceFrameFrame` 😅

Reply:
I renamed it to default device framw